### PR TITLE
Create blog.html

### PR DESCRIPTION
**Pull request by SakibvHossain**

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="Euzan - Creative Portfolio Template">
+        <meta name="keywords" content="Html, Css, jQuery, JavaScript, Euzan, responsive, onepage, personal, Portfolio, design, creative, template">
+        
+        <!-- Title -->
+        <title>Euzan - Creative Portfolio Template</title>
```
Comment (anika-as) [inline blog.html]: Give a valid name here

Comment (SakibvHossain) [reply blog.html]: Provided valid name please check it out

Comment (SakibvHossain) [reply blog.html]: Well done

Comment (anika-as) [reply blog.html]: Well done

**Review by anika-as — APPROVED** [review]

**Pull request merged**